### ISSUE TITLE
Edit on Github button seems to be broken

### ISSUE THREAD

**Issue by shcheklein** [iterative/dvc.org#3243]
I detected this trough the Hexometer monitoring:

<img width="603" alt="Screen Shot 2022-01-31 at 6 26 34 PM" src="https://user-images.githubusercontent.com/3659196/151904596-0bdf2244-0e11-4d7b-8860-558e51864497.png">

Let's do a postmortem and analyze how could we improve this. Seems like the right way is to pay more attention to the link checker ? we need to see if it was able to detect this in the first place.

**Comment by shcheklein** [comment]
<img width="583" alt="Screen Shot 2022-01-31 at 6 28 52 PM" src="https://user-images.githubusercontent.com/3659196/151904718-74652a44-9ea3-4b1a-a80a-4899d48a8644.png">

**Comment by rogermparent** [comment]
Thanks @julieg18 for the fix! This was wholly on me because I made a typo in moving the logic for generating GitHub links: I set the repo as `iterative/dvc` instead of `iterative/dvc.org`

> Seems like the right way is to pay more attention to the link checker ?

The PR link checker wouldn't have caught this, as it only inspects sources, and I'm pretty sure the whole-site checker does as well. The resulting links from these "edit on GitHub" buttons are only ever put together dynamically at the end of the process with a template string, so the link checker never reads them.
